data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -0,0 +1,758 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "cudgel_plus_one",
+    "copy-from": "cudgel",
+    "looks_like": "cudgel",
+    "name": "cudgel +1",
+    "proportional": { "price": 3.0, "bashing": 1.1, "cutting": 1.1 },
+    "relative": { "to_hit": 1 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "cudgel_plus_two",
+    "copy-from": "cudgel",
+    "looks_like": "cudgel",
+    "name": "cudgel +2",
+    "proportional": { "price": 3.0, "bashing": 1.1, "cutting": 1.1 },
```

Comment:
```suggestion
    "proportional": { "price": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": .8 },
```

Reply:
Damn, I should have caught that, it was the first weapon I did, but I only did the +1 version and went back and did the +2 at the end.